deployments/tools/export/lakefs_export.py

```diff
@@ -13,8 +13,10 @@
 LAKEFS_ACCESS_KEY = os.getenv('LAKEFS_ACCESS_KEY_ID')
 LAKEFS_SECRET_KEY = os.getenv('LAKEFS_SECRET_ACCESS_KEY')
 LAKEFS_ENDPOINT = os.getenv('LAKEFS_ENDPOINT')
-S3_ACCESS_KEY = os.getenv('AWS_ACCESS_KEY_ID')
-S3_SECRET_KEY = os.getenv('AWS_SECRET_ACCESS_KEY')
+S3_PROVIDER = os.getenv('S3_PROVIDER', default="AWS")
+S3_ACCESS_KEY = os.getenv('S3_ACCESS_KEY_ID')
+S3_SECRET_KEY = os.getenv('S3_SECRET_ACCESS_KEY')
+S3_ENDPOINT = os.getenv('S3_ENDPOINT_URL', default="")
```

Comment:
Unfortunately this breaks backwards compatibility: someone using the script will need to change their environment variables in order to keep on using it.  The new script version must accept the old values if the new ones are missing.